app/Http/Controllers/admin/SettingsController.php

```diff
@@ -80,15 +80,17 @@ public function updateAppearance(Request $request)
   {
       $this->validate($request, $this->rules);
 
-      $input = Reqeust::only(
+      $input = Input::only(
```

Comment:
Since it looks like an instance of the Request is being passed in to this function, could you do `$request->only(...)` here? If not, I'd probably vote for using the `Request` facade (sans the original typo 😉) since `Input` is just an alias for that.